projects/plugins/jetpack/extensions/blocks/contact-form/components/jetpack-field-controls.js

```diff
@@ -18,13 +21,86 @@ import JetpackFieldWidth from './jetpack-field-width';
 import JetpackManageResponsesSettings from './jetpack-manage-responses-settings';
 
 const JetpackFieldControls = ( {
-	setAttributes,
-	width,
+	attributes,
 	id,
-	required,
 	placeholder,
 	placeholderField = 'placeholder',
+	required,
+	setAttributes,
+	width,
+	type,
 } ) => {
+	const setNumberAttribute = ( key, parse = parseInt ) => value => {
+		const parsedValue = parse( value, 10 );
+
+		setAttributes( {
+			[ key ]: ! isNaN( parsedValue ) ? parsedValue : '',
+		} );
+	};
+
+	let colorSettings = [];
+	let borderWidthControls = '';
+	let borderRadiusControls = '';
+
+	if ( typeof type === 'string' && ( type === 'radio' || type === 'checkbox' ) ) {
```

Comment:
Hmm, can we maybe put this into a variable so it's more obvious what it's about?
Like `const hasBorderControls =  type === 'radio' || type === 'checkbox';`.

The `typeof` check is also a little overkill.

Reply:
type is mostly undefined AFAICT, which is why I added the typeof, but happy to omit it. I'll test that.